src/nimble_kokkos.cc

```diff
@@ -705,7 +663,7 @@ int ExplicitTimeIntegrator(nimble::Parser & parser,
 
     // Perform a reduction to obtain correct values on MPI boundaries
     watch_internal.push_region("MPI reduction");
-    myVectorCommunicator.VectorReduction(mpi_vector_dimension, internal_force_h);
+    myVectorCommunicator->VectorReduction(mpi_vector_dim, internal_force_h);
```

Comment:
Use a constant variable for 'mpi_vector_dim'. 
Ideally we should use the spatial dimension but, currently, 'internal_force_h' (and others) are double*[3] (or a Kokkos equivalent).

Reply:
@rjones-sandia Comment to address https://github.com/NimbleSM/NimbleSM/pull/159#pullrequestreview-605323423